public/sitemap.xml

```diff
@@ -0,0 +1,27 @@
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
+<url>
+<loc>https://chitchatter.im/</loc>
+<lastmod>2023-03-08T15:39:35+00:00</lastmod>
```

Comment:
It seems that these values would have to be kept up to date with each deployment. This isn't something that should be done manually, so we should have some automated process within [the deploy process](https://github.com/jeremyckahn/chitchatter/blob/develop/.github/workflows/deploy.yml) that performs this work for us. @MohammadTaseenKhan would you be willing to change this PR to generate an updated `sitemap.xml` at build time?

Reply:
more information about it

https://yoast.com/what-is-an-xml-sitemap-and-why-should-you-have-one

Reply:
> well time difference in sitemap.xml does not effect the google SEO

That may be the case, but we shouldn't be knowingly publishing inaccurate information. There may be undesirable results we're not considering as a result of that, and in which case we may be better served by not publishing a `sitemap.xml` file at all. I think this needs to be sorted out before this PR can be merged.

Do you think it would be a significant effort to change this PR such that it generates the `sitemap.xml` file with accurate `lastmod` timestamps at build time?

Reply:
ok then i will update the sitemap time lastmod after every build

To know realtime build i need to add webhook to your github repo 

discord webhook is ok i think

I have mailed you the webhook url

Reply:
> ok then i will update the sitemap time lastmod after every build

I'd really rather not introduce a manual process for each app build/deployment, regardless of who is willing to take responsibility for doing the work. It appears that this work can be automated, and we'd be well-served by making the upfront time investment to do so.

> To know realtime build i need to add webhook to your github repo
> 
> discord webhook is ok i think

That approach seems a little more involved than what's necessary to automate this. A simple way to get this done would be to modify the `build` script to generate the `sitemap.xml`:

https://github.com/jeremyckahn/chitchatter/blob/442f5406ec26d001798c1a2b5a5d9a05fdc77459/package.json#L54

What do you think of this approach?

Reply:
looks great to me

Reply:
Dont forget to Ping Google

ping Google about the uploaded sitemaps

```js
asyncfunctionpingGoogle({siteURL, sitemapIndex}) {  
  console.log('Pinging Google sitemap has been updated...');  
  await when.promise(function(resolve, reject, notify) {    
    rest      
      .get('http://google.com/ping?sitemap=' + siteURL + '/' + sitemapIndex)      
      .on('success', function(data, response) {        
        console.log('Google Ping: ' + data);                resolve();      
      })      
      .on('fail', function(data, response) {        
        console.log('Google Ping Error:', data);                resolve();      
      });  
  });  
console.log('Google pinged.');}
```

Reply:
Good idea! Thanks for sharing this info @MohammadTaseenKhan. Is this something you were planning to address yourself, or would you like someone else to take it over from here?

Reply:
this is the code that my teacher shown to me that we can also ping google that we have updated our updated sitemap

it can be also done with link or like sponsored links